styx-cli/pom.xml

```diff
@@ -69,6 +69,11 @@
       <artifactId>jackson-annotations</artifactId>
       <version>${jackson.version}</version>
     </dependency>
+      <dependency>
+          <groupId>com.spotify</groupId>
+          <artifactId>styx-client</artifactId>
+          <version>1.0.6-SNAPSHOT</version>
```

Comment:
We should probably starting using `<version>${project.version}</version>` for inter-module dependency. Or even move to `dependencyManagement`. Maybe a separated PR. However things like `<version>${jackson.version}</version>` in this pom.xml should be gone.